DominicGBauer/information-retrieval
# chore: integration and refactor

# Pull Request
Code refactor and functionality update 

## Scope
New function for integration, cleaned up files 

## Work Done
Refactor from Marc's previous code. Added a new function to integrate Alex's code. Cleaned up develop branch to remove unnecessary files. 

## Steps to Test
Create a JSON file with your UCT username and password in the code directory called uctpw.json. Run main and uncomment relevant testing functions as you see fit.
